sympy/solvers/tests/test_solvers.py

```diff
@@ -1141,12 +1143,14 @@ def test__invert():
 
 def test_issue_4463():
     assert solve(-a*x + 2*x*log(x), x) == [exp(a/2)]
-    assert solve(a/x + exp(x/2), x) == [2*LambertW(-a/2)]
+    #assert solve(a/x + exp(x/2), x) gives [2*LambertW(-a/2), 2*LambertW(a/2)]
```

Comment:
I am getting a difiiculty here to eliminate `2*LambertW(a/2)` from solution.

Reply:
But, isn't the solution without `2*LambertW(a/2)` inclomplete?
After the patch, isn't it supposed to give the complete solution.

Reply:
@Sc0rpi0n101 `[2*LambertW(-a/2)]` is the complete solution. Checked from [wolfram](https://www.wolframalpha.com/input/?i=solve+a%2Fx+%2B+exp(x%2F2)+for+x)